### DISCUSSION TITLE
How to bind a key to "enable mouse look" option?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13841] [Ideas and suggestions]
The "enable mouse look" setting is sometimes convenient for looking into the distance in caves as example. Most of the time it causes an unpleasant wobble on the screen. Is it possible to make  "enable mouse look" turned on and off using a console command and bind a key to the command.

**Comment by SomeRandomNoobKekeke** [comment]
https://steamcommunity.com/sharedfiles/filedetails/?id=2817020588

**Reply by pavel4win** [reply]
thank you, I will try this mod.
